resources/translations/fi.edn

```diff
@@ -141,7 +141,9 @@
   :cart {:add "Lisää koriin"
          :apply "Hae"
          :apply-for-bundle "Hae näitä aineistoja (%1 kpl) yhdessä "
+         :empty [:div.intro [:p "Valitse resursseja haettavaksi."]]
          :header "%1 aineistoa korissa"
+         :intro [:div.intro [:p "Tässä ovat kaikki resurssit jotka olet valinnut. Resurssit jotka voi hakea samalla hakemuksella ryhmitellään automaattisesti yhteen."]]
```

Comment:
"Tässä ovat kaikki resurssit jotka olet valinnut." -> "Tässä ovat kaikki resurssit, jotka olet valinnut."

A proposal: "Resurssit jotka voi hakea samalla hakemuksella ryhmitellään automaattisesti yhteen." -> "Samalla hakemuksella haettavat resurssit ryhmitellään automaattisesti yhteen."

Formulating it like this would remove the need for a comma before and after "jotka voi hakea samalla hakemuksella" (so-called "kiilalause").

Reply:
I think I got a much better one now.
> Tässä ovat valitsemasi resurssit. Samalla hakemuksella haettavat resurssit ryhmitellään automaattisesti yhteen.